prod-refractr.yml

```diff
@@ -1313,3 +1313,17 @@ refracts:
 - mozilla.org/ro:
   - mozilla.ro
   - www.mozilla.ro
+
+# SE-4288
+- nginx: |
+    server {
+      server_name mozilla.social;
+
+      location / {
```

Comment:
This would only return 410 for the root path, right? That wouldn't mean much actually and not needed, rather it should be `/users/*` that should return 410 because this is the path the AP servers would query about.

Reply:
Ah, cool, match-all sounds fine. Thanks!